src/main/java/io/strimzi/kafka/bridge/converter/MessageConverter.java

```diff
@@ -32,18 +32,20 @@
 	 *
 	 * @param kafkaTopic Kafka topic for sending message
 	 * @param message message to convert
+	 * @param partition partition of topic where the messages are sent when partition is specified in the URL
```

Comment:
URL doesn't make sense in case of AMQP and this is a generic interface for AMQP and HTTP or other in the future. So just saying "partition to send the message to when it's not specified in the message itself"